packages/main/src/MultiInput.ts

```diff
@@ -356,6 +327,10 @@ class MultiInput extends Input {
 		return this.shadowRoot!.querySelector<Tokenizer>("[ui5-tokenizer]")!;
 	}
 
+	get tokenizerExpanded() {
+		return this.tokenizer && this.tokenizer.expanded;
+	}
+
 	get _tokensCountText() {
```

Comment:
// can we use object e.g. { 0: TOKENIZER_ARIA_CONTAIN_TOKEN, 1: TOKENIZER_ARIA_CONTAIN_ONE_TOKEN } and query it

Reply:
Done.